GAD2.py

```diff
@@ -0,0 +1,209 @@
+import os
+import re
+from typing import TextIO, Optional, Union
+from abc import ABC, abstractmethod
+from Bio import SeqIO, SeqUtils
```

Comment:
Правильно остортировано по происхождению бибилиотек, но нет лексикографического порядка.
```suggestion
import os
import re
from abc import ABC, abstractmethod
from typing import TextIO, Optional, Union
from Bio import SeqIO, SeqUtils
```